Parsers/ASimProcessEvent/Parsers/vimProcessCreateTrendMicroVisionOne.yaml

```diff
@@ -0,0 +1,236 @@
+Parser:
```

Comment:
No TerminateProcess support in TrendMicro?

Reply:
@vakohl 
As discussed earlier with Devika, we can continue with only Process Creation parser.
Yes there is event exists as per doc for Process Termination but we do not have any live or sample data for that event that is why we haven't included.